### DISCUSSION TITLE
Same Code with free $200 credit, transcription works for 2 min and on next minute it connects, but no transcription comes bacl

### DISCUSSION THREAD

**Discussion by Rajan-Gupta1** [deepgram/community#667] [General help]
I am using the deepgram/SDK and not using websocket directly
Here is the request ID:
{
    "type": "Metadata",
    "transaction_key": "deprecated",
    "request_id": "35733be7-505b-4fd4-b3f8-1b79c8c3083a",
    "sha256": "8100f8976e8e97a0f471b2956629ccff4b90c86d0d9f2146b5feafe76b65c0db",
    "created": "2024-03-30T00:02:54.910Z",
    "duration": 0,
    "channels": 0
}
Can you please tell why using the same key it transcribes live audio and right after it it does not work.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by Rajan-Gupta1** [comment]
The problem was I was not sending keepalive(). Once I started to send every 5 seconds it started to work.

**Comment by Rajan-Gupta1** [comment]
Reopening since even with KeepAlive it stops working after two times. Is there a limitation on the free$200 credit account on how many times it can be used in an hour  before it stops sending transcripts

**Reply by Rajan-Gupta1** [reply]
{
    "type": "Metadata",
    "transaction_key": "deprecated",
    "request_id": "ae965a8c-ad48-4ed9-9bcf-da64d8754fc8",
    "sha256": "8b617ec4cf0c0845856abff069892568771bc465107cc63ee5ceb27727cca9ac",
    "created": "2024-03-30T20:05:05.911Z",
    "duration": 0,
    "channels": 0
}

**Comment by jpvajda** [comment]
@Rajan-Gupta1  I don't believe this has anything to do with your free credits.  For streaming the use of keep alive can be a bit tricky so we recently released a[ detailed guide](https://developers.deepgram.com/docs/keep-alive) on that functionality. Check it out and see if it helps you. 

Also Streaming requests do have specific [rate limits](https://developers.deepgram.com/reference/api-rate-limits) you should be aware of as well. 

Also for us to go deeper on this issue, if you can provide some code samples of what you are using that would help us troubleshoot.
